README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+## hw01
+
+## Regina Catipon Bio
+
+
+<div><img src="https://rkcatipon.files.wordpress.com/2015/02/10014802_10153266423432857_4524792863877171598_o.jpg" width="200" height="300" align="left"></div>
```

Comment:
Nice use of Image!

Reply:
If you want the image aligned with the text, then yes you have to use HTML code. Markdown syntax is limited in its capabilities (intentionally so). Otherwise this section could look like

```
## Regina Catipon Bio

![Regina](https://rkcatipon.files.wordpress.com/2015/02/10014802_10153266423432857_4524792863877171598_o.jpg)

Hi, I’m **Regina**. I’m one part verve and two parts curiosity.

A former journalist turned social scientist, I am now  pursuing a masters in Computational Social Science from the University of Chicago to explore information dissemination online.
```

## Regina Catipon Bio

![Regina](https://rkcatipon.files.wordpress.com/2015/02/10014802_10153266423432857_4524792863877171598_o.jpg)

Hi, I’m **Regina**. I’m one part verve and two parts curiosity.

A former journalist turned social scientist, I am now  pursuing a masters in Computational Social Science from the University of Chicago to explore information dissemination online.

Reply:
Also note you cannot control the image size using Markdown. You'd have to use your HTML syntax, or manually resize the image first